3-networks/modules/standard_shared_vpc/firewall.tf

```diff
@@ -110,6 +112,10 @@ resource "google_compute_firewall" "allow_windows_activation" {
     protocol = "tcp"
     ports    = ["1688"]
   }
+
+  destination_ranges = ["35.190.247.13/32"]
+
+  target_tags = ["allow-win"]
```

Comment:
I think something more specific like `allow-win-kms` or `allow-win-activation` would be better for this

Reply:
I will user `allow-win-activation`